README.md

```diff
@@ -2,39 +2,32 @@
 
 App for tagging content on GOV.UK.
 
-## Nomenclature
+## Screenshots
 
-- **Word**: definition of word, and how it's used in the code
+![Lookup form](docs/screenshot-1.png)
+![Tagging interface](docs/screenshot-2.png)
 
 ## Technical documentation
 
-This is a Ruby on Rails application that works with the publishing-api to manage
-the "links hash" for content on GOV.UK.
+This is a Ruby on Rails application that works with the publishing-api to manage the "links hash" for content on GOV.UK.
```

Comment:
I could not find any doc that covers that the links hash is but if there's one `links hash` could link to it.